src/main/java/com/chessmates/lichess/data/LichessApi.groovy

```diff
@@ -11,15 +7,15 @@ interface LichessApi {
     /**
      * Get the players of a given Lichess team.
      */
-    LichessResultPage<Player> getPlayers(String teamId, int pageNumber, int pageSize)
+    def getPlayers(String teamId, int pageNumber, int pageSize)
 
     /**
      * Get games of a given player.
      */
-    LichessResultPage<Game> getGames(String playerId, int pageNumber, int pageSize)
+    def getGames(String playerId, int pageNumber, int pageSize)
```

Comment:
Agreed - this needs to be renamed to make it clear that we're returning a page of data here, to differentiate it from the getGames methods that return a list of games.